lib/qam.pm

```diff
@@ -124,7 +124,7 @@ sub ssh_add_test_repositories {
 sub remove_test_repositories {
 
     type_string 'repos=($(zypper lr -e - | grep "name=TEST|baseurl=ftp" | cut -d= -f2)); if [ ${#repos[@]} -ne 0 ]; then zypper rr ${repos[@]}; fi';
-    type_string "\n";
+    send_key 'ret';
```

Comment:
It'd be nice if `send_key 'ret'` could be replaced just with `enter_cmd` without any param.

Reply:
Well, I would prefer rather not. We introduced the new function based on a request by users to make something more explicit what already worked in before.

But if you promise to keep you a secret I can tell you: (*whispering*) enter_cmd without parameters already works! ;)